.github/workflows/pythonapp.yml

```diff
@@ -138,19 +138,19 @@ jobs:
         cd ~/work/hummingbird/tvm/python
         python3 -m pip install -e .
 
-    # We don't run pytest for macos py3.9 since we do coverage for that case.
+    # We don't run pytest for ubuntu py3.9 since we do coverage for that case.
```

Comment:
Now there is some problem with macos + codecov that is blocking the pipeline.  Changing this to unblock, and also since mac is slowest it makes no sense to make it worse! Will fix additional issues in separate PR.